cli/src/main/resources/hudson/cli/client/Messages.properties

```diff
@@ -6,6 +6,7 @@ CLI.Usage=Jenkins CLI\n\
    -p HOST:PORT : HTTP proxy host and port for HTTPS proxy tunneling. See http://jenkins-ci.org/https-proxy-tunnel\n\
    -noCertificateCheck : bypass HTTPS certificate check entirely. Use with caution\n\
    -noKeyAuth   : don't try to load the SSH authentication private key. Conflicts with -i\n\
+   -v           : verbose output. Display logs on the console\n\
```

Comment:
I think it should briefly explain the behavior and the j.u.l logging levels to be set